Mixpanel/Mixpanel.h

```diff
@@ -412,24 +412,24 @@ extern NSString *const MPNotificationTypeTakeover;
 
  If you are not using the AdSupport Framework (iAds), then we use the
  <code>[UIDevice currentDevice].identifierForVendor</code> (IFV) string as the
- default distinct ID.  This ID will identify a user across all apps by the same
+ default local distinct ID.  This ID will identify a user across all apps by the same
  vendor, but cannot be used to link the same user across apps from different
  vendors.
 
  If you are showing iAds in your application, you are allowed use the iOS ID
  for Advertising (IFA) to identify users. If you have this framework in your
- app, Mixpanel will use the IFA as the default distinct ID. If you have
+ app, Mixpanel will use the IFA as the default local distinct ID. If you have
  AdSupport installed but still don't want to use the IFA, you can define the
  <code>MIXPANEL_NO_IFA</code> preprocessor flag in your build settings, and
- Mixpanel will use the IFV as the default distinct ID.
+ Mixpanel will use the IFV as the default local distinct ID.
 
  If we are unable to get an IFA or IFV, we will fall back to generating a
  random persistent UUID. If you want to always use a random persistent UUID
  you can define the <code>MIXPANEL_RANDOM_DISTINCT_ID</code> preprocessor flag
  in your build settings.
 
  For tracking events, you do not need to call <code>identify:</code> if you
- want to use the default.  However, <b>Mixpanel People always requires an
+ want to use the default.  However, <b>Mixpanel User profiles always require an
```

Comment:
remove " if you want to use the default."

Reply:
(as context: with id merge enabled, you can't control the resolved distinct_id -- you used to be able to which is why the sentence was worded this way)